Allow specifying a custom subscription as a consumer endpoint (#262)
GoogleCloudPlatform/spring-cloud-gcp: docs/src/main/asciidoc/spring-stream.adoc

> If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.

Reviewer: I think we should add a note here that, only anonymous subscriptions will be cleaned up.

Author: It's mentioned twice below this.

> The topic name will be the same as the destination name, and the subscription name will be the destination name followed by the consumer group name.

Reviewer: Since you talk about the subscription name below, maybe take out the second part of the sentence?

Author: removed

> Regardless of the `auto-create-resources` setting, you can control the subscriber with the following options (in order of precedence):

Reviewer: ```suggestion
Regardless of the `auto-create-resources` setting, the subscription name will be determined as follows (in order of precedence):
```

Author: Ended up removing this text block

> * A pre-existing subscription (use `spring.cloud.stream.gcp.pubsub.bindings.{CONSUMER_NAME}.consumer.subscriptionName`)

Reviewer: ```suggestion
* If specified, `spring.cloud.stream.gcp.pubsub.bindings.{CONSUMER_NAME}.consumer.subscriptionName` property
```

Author: With the "in order of precedence" above, I don't think it's necessary to have to say "if specified"

> * A consumer group that uses the topic name (use `spring.cloud.stream.bindings.events.group` to create a subscription named `<topicName>.<group>`)

Reviewer: ```suggestion
* If specified, `spring.cloud.stream.bindings.events.group` property that results in a subscription named `<topicName>.<group>`
```
